index.html

```diff
@@ -17,10 +18,126 @@
+                        <button class="item" type="button">JavaScript</button>
+                    </div>
+                    <button class="project" type="button">see this project<i class="fa-solid fa-right-long"></i></button>
+                </div>
+            </section>
+            <section class="layout-2">
+                <div class="wrapper">
+                    <img src="img/laptop2.svg" alt="project4">
+                    <h2>Project Name goes there</h2>
+                    <div class="tech-button">
+                        <button class="item" type="button">HTML/CSS</button>
+                        <button class="item" type="button">Ruby on Rails</button>
+                        <button class="item" type="button">JavaScript</button>
+                    </div>
+                    <button class="project" type="button">see this project<i class="fa-solid fa-right-long"></i></button>
+                </div>
+            </section>
+            <section class="layout-2">
+                <div class="wrapper">
+                    <img src="img/laptop3.svg" alt="project5">
+                    <h2>Project Name goes there</h2>
+                    <div class="tech-button">
+                        <button class="item" type="button">HTML/CSS</button>
+                        <button class="item" type="button">Ruby on Rails</button>
+                        <button class="item" type="button">JavaScript</button>
+                    </div>
+                    <button class="project" type="button">see this project <i class="fa-solid fa-right-long"></i></button>
+                </div>
+            </section>
+            <section class="layout-2">
+                <div class="wrapper">
+                    <img src="img/laptop4.svg" alt="project6">
+                    <h2>Project Name goes there</h2>
+                    <div class="tech-button">
+                        <button class="item" type="button">HTML/CSS</button>
+                        <button class="item" type="button">Ruby on Rails</button>
+                        <button class="item" type="button">JavaScript</button>
+                    </div>
+                    <button class="project" type="button">see this project<i class="fa-solid fa-right-long"></i></button>
+                </div>
+            </section>
+            <section class="about">
+                <h1>About</h1>
+                <p>Hello I'm a software developer! I can help you build a product , 
+                    feature or website Look through some of my work
+                    and experience! If you like what you see and have a project you need coded, 
+                    don't hestiate to contact me.
+                </p>
+                <button class="get-cv" type="button">Get my CV<i class="fa-solid fa-arrow-down-to-bracket"></i></button>
+                <div class="technicals">
+                    <img src="img/languages.jpg" alt="languages">
+                    <h3>Languages</h3>
```

Comment:
- [ ] The headings Languages, Frameworks, skills have bigger font size, Please reduce the font size for these headings and make sure it matches the design on the template.